alignak/dependencynode.py

```diff
@@ -151,20 +151,20 @@ def get_state(self, hosts, services):
             return self.get_host_node_state(host.last_hard_state_id,
                                             host.problem_has_been_acknowledged,
                                             host.in_scheduled_downtime)
-        elif self.operand == 'service':
```

Comment:
It's not pylint want, It only say, not use: 
```
if self.operand == 'host':
    return 'yes'
else:
    return 'no'
```
but use better:
```
if self.operand == 'host':
    return 'yes'
return 'no'
```
So you can use if and elif but remove the else when have return in if or elif